includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -74,13 +74,19 @@ function get_columns() {
 
 			case 'users':
 			default:
-				$columns = array(
-					'title'             => __( 'Student', 'sensei-lms' ),
+				$students_header_total_values = $this->set_users_table_header_total_values();
+				$students_count               = is_array( $this->items ) ? count( $this->items ) : ! empty( $this->items );
```

Comment:
I think you can use `$this->total_items` to get the count.

Reply:
@donnapep, are you sure it's not working when exporting. I did the similar thing you've done, moved the get_colums call after fetching row data: [here](https://github.com/Automattic/sensei/pull/4865/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR276) When I tested it it was working. Here is the image of the export: 

<img width="940" alt="image" src="https://user-images.githubusercontent.com/7208249/157873731-0751ca5f-9ed1-455e-b26a-35910f0e438e.png">

Can you try again, since I'm afraid that if you don't get the value it means that this approach (the same you used) is not working always